src/main/java/api/StoreDistance.java

```diff
@@ -0,0 +1,17 @@
+package api;
+
+import heb.Store;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class StoreDistance {
```

Comment:
It's been a while since I did much Jackson-ing. There's probably a more elegant way to marshal the API response without having to introduce a container class.

<img width="656" alt="H‑E‑B_Curbside___Grocery_Delivery___Austin__Houston___More___HEB_com" src="https://user-images.githubusercontent.com/15816/77613099-dd7aa000-6ef7-11ea-869e-95e7d0998da8.png">